http-api/src/lib.rs

```diff
@@ -450,8 +466,20 @@ async fn history_handler(
 
     let commits = browse(reference, ctx.paths, |browser| {
         let mut result = radicle_source::commits::<PeerId>(browser, None)?;
-        let headers: Vec<Header> = result
+
+        // To create chunks that make sense we sort the commits from newest to oldest.
+        result
             .headers
+            .sort_by(|a, b| b.committer_time.cmp(&a.committer_time));
+        let paginated_headers: Vec<Vec<Header>> = result
+            .headers
+            .chunks(per_page.unwrap_or_default())
```

Comment:
This will return `0` when `None`, because `0` is the default of `usize`

Reply:
Ok nevermind you're right :+1: